hipaa_final_omnibus_security_rule_2013/164_314/164_314_b_2.sp

```diff
@@ -27,7 +27,7 @@ benchmark "hipaa_final_omnibus_security_rule_2013_164_314_b_2" {
     control.s3_public_access_block_account,
     control.sagemaker_endpoint_configuration_encryption_at_rest_enabled,
     control.sagemaker_notebook_instance_encryption_at_rest_enabled,
-    control.vpc_endpoint_service_acceptance_required_enabled
+    control.vpc_configured_to_use_vpc_endpoints
```

Comment:
@Priyanka-Chatterjee-2000 Please add tag `hipaa_final_omnibus_security_rule_2013 = "true"` to the control and check for other added controls as well.